docs/design/basic_gateway/design.md

```diff
@@ -0,0 +1,411 @@
+## On-demand installing of table entries based on basic gateways
```

Comment:
Is this "basic gateways" different from bouncers? If so can you detail their function? if not, can you just stick to the terminology?

Reply:
Okay, “basic gateways” and “bouncers” are really different.  We will update more details in later documents.

Reply:
Sure thanks. would you please make sure to address all questions in the next revision?